### PR TITLE
Start xJoelituh

### PR DESCRIPTION
**Pull request by SSlayer9**

### CODE REVIEW

**Comment by SSlayer9** [comment]
Item is probably messed up. Contacted xJoel about something will update when they respond.

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -2080,6 +2080,37 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
 	},
+		
+	// xJoelituh
+	burnbone: {
+		accuracy: 90,
+		basePower: 0,
+		category: "Status",
+		desc: "",
+		shortDesc: "",
+		name: "Move Name",
```
Comment (ghost) [inline data/mods/ssb/moves.ts]:
```suggestion
		name: "Burn Bone",
```

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -2080,6 +2080,37 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+		flags: {protect:1},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Will-O-Wisp', target);
+			this.add('-anim', source, 'Shadow Bone', target);
+		},
+		onHit(target, source) {
+			if (target.status && target.status === 'brn') return;
```
Comment (ghost) [inline data/mods/ssb/moves.ts]:
```suggestion
			if (!target.trySetStatus('brn', source, move)) return;
```

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -2080,6 +2080,37 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+			source.heal(source.baseMaxHp / 3);
```
Comment (xInstruct) [inline data/mods/ssb/moves.ts:2104]:
```suggestion
			this.heal(source.baseMaxHp / 3);
```

**Review by ghost — CHANGES_REQUESTED** [review]
Add descriptions to the move and item and then Ill merge it after further review.

**Review by ghost — APPROVED** [review]

File: data/mods/ssb/random-teams.ts [hunk 4]
```diff
@@ -382,6 +382,12 @@ export const ssbSets: SSBSets = {
 		signatureMove: 'DRIP BAYLESS',
 		evs: {hp: 252, spa: 252, def: 4}, ivs: {atk: 0}, nature: 'Modest',
 	},
+	xJoelituh: {
+		species: 'Marowak-Alola', ability: 'Mountaineer', item: 'Rare Bone', gender: 'M',
+		moves: ['Poltergeist', 'Fire Punch', 'Stomping Tantrum'],
+		signatureMove: 'Burn Bone',
+		evs: {hp: 248, atk: 8, spd: 252}, nature: 'Careful', shiny: true
```
Comment (ghost) [inline data/mods/ssb/random-teams.ts]:
```suggestion
		evs: {hp: 248, atk: 8, spd: 252}, nature: 'Careful', shiny: true,
```

File: data/mods/ssb/moves.ts [hunk 5]
```diff
@@ -2159,6 +2159,36 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Will-O-Wisp', target);
+			this.add('-anim', source, 'Shadow Bone', target);
+		},
+		onHit(target, source, move) {
+			if (target.trySetStatus('brn', source, move)) {
+				source.heal(source.baseMaxHp / 3);
```
Comment (ghost) [inline data/mods/ssb/moves.ts]:
```suggestion
				source.heal(source.baseMaxhp / 3);
```

**Pull request merged**